python/ray/tune/tune.py

```diff
@@ -706,6 +741,16 @@ class and registered trainables.
 
     progress_metrics = _detect_progress_metrics(_get_trainable(run_or_experiment))
 
+    # setting the right context for console logging
+    is_single_trial = search_alg.total_samples == 1
+
+    def is_rllib():
+        from ray.rllib.algorithms.algorithm import Algorithm
```

Comment:
you are not gonna really import the Algorithm class, are you?
at some point we were saying the logging for individual RLlib / Train trials will happen in RLlib / Train repo.
if we really need to get this bit in Tune, I suggest we look at https://github.com/ray-project/ray/blob/master/rllib/algorithms/registry.py#L225-L261